### PR TITLE
Stop using threads subsystem in testsuite

### PR DESCRIPTION
**Pull request by ctomc**

### CODE REVIEW

**Comment by stuartwdouglas** [comment]
Failures look related

**Comment by ctomc** [comment]
yeah they are :|

File: pom.xml [hunk 1]
```diff
@@ -220,7 +220,7 @@
         <version.org.testng>6.1.1</version.org.testng>
         <version.org.wildfly.build-tools>1.0.0.Alpha8</version.org.wildfly.build-tools>
         <version.org.wildfly.checkstyle-config>1.0.1.Final</version.org.wildfly.checkstyle-config>
-        <version.org.wildfly.core>1.0.0.Alpha18</version.org.wildfly.core>
+        <version.org.wildfly.core>1.0.0.Alpha19-SNAPSHOT</version.org.wildfly.core>
```
Comment (bstansberry) [inline pom.xml]: Why is a core bump needed for this?

Comment (ctomc) [inline pom.xml]: mistake, I needed it locally to test it with latest core that has threads subsystem deprecated
looks like it got in with rebase...

Comment (ctomc) [inline pom.xml]: fixed

**Pull request merged**